package.json

```diff
@@ -1,5 +1,8 @@
 {
+	"scripts": {
+		"demo": "obt demo && http-server demos/local"
```

Comment:
We have a flag which will run a server -- `obt demo --run-server`

Reply:
Oh perfect! I'll delete the script 😄